include/grpc/impl/codegen/port_platform.h

```diff
@@ -27,6 +27,13 @@
  *  - some syscalls to be made directly
  */
 
+// [[deprecated]] attribute is only available since C++14
+#if __cplusplus >= 201402L
+#define GRPC_DEPRECATED(reason) [[deprecated(reason)]]
```

Comment:
Doesn't look like you're actually using this for the experimental API.

Reply:
Thanks for noticing! Fixed it!